modules/lib/src/main/resources/assets/js/app/inputtype/ui/text/dialog/LinkModalDialog.ts

```diff
@@ -299,30 +291,15 @@ export class LinkModalDialog
         ]);
     }
 
-    private createAnchorPanel(anchorList: string[]): Panel {
-        return this.createFormPanel([
-            this.createAnchorDropdown(anchorList)
-        ]);
+    private static validationAlwaysValid(): string {
+        return undefined;
     }
 
-    private createAnchorDropdown(anchorList: string[]): FormItem {
-        const dropDown = new Dropdown<string>('anchor', <DropdownConfig<string>>{});
+    private validationQueryParams(): string {
+        const isValid = Array.from(this.paramsFormItem.getHTMLElement().getElementsByTagName('input'))
+            .every((input: HTMLInputElement) => input.value !== '');
 
-        anchorList.forEach((anchor: string) => {
-            dropDown.addOption(Option.create<string>()
-                .setValue(LinkModalDialog.anchorPrefix + anchor)
-                .setDisplayValue(anchor)
-                .build());
-        });
-
-        if (this.getAnchor()) {
-            dropDown.setValue(this.getAnchor());
-        }
-
-        const formItemBuilder = new ModalDialogFormItemBuilder('anchor', i18n('dialog.link.tabname.anchor')).setValidator(
-            Validators.required).setInputEl(dropDown);
-
-        return this.createFormItem(formItemBuilder);
```

Comment:
Can't remove this. We have a "hidden" feature for adding links to anchors on the same page. Can be tested as follows:
1. Use Anchor button (the one with the flag) in the editor toolbar to insert an anchor into the HTML text
2. Open the Link dialog.
3. You will see the "Anchor" tab and you can select your anchor from the dropdown to link to it.

![image](https://user-images.githubusercontent.com/11612490/174996500-e0ce209d-a561-438b-8a9a-273f451ef719.png)

![image](https://user-images.githubusercontent.com/11612490/174996665-eaf2b549-03e3-4312-9636-5276409af4e1.png)